lib/timers.js

```diff
@@ -160,16 +160,51 @@ function listOnTimeout() {
   debug('timeout callback %d', msecs);
 
   var now = TimerWrap.now();
-  debug('now: %s', now);
+  debug('now: %d', now);
 
-  var diff, timer;
+  var diff, timer, timeRemaining;
```

Comment:
Please just define it inline, we've been moving away from these old "optimizations" -- which really don't matter in todays optimizing JITs